resources/themes/cura-light/icons/low/CheckMark.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Why not use Check.svg? 

I think it's rather confusing to have both CheckMark and Check in the icon set, both which only have very subtle differences.

Reply:
Do plugins use this icon? The CheckMark.svg width/height is 2x Check.

Reply:
If they do, we can add that to the deprecated icon names (with the replacement) or even just remove them since the next update is a major SDK update.